.github/actions/deploy-dataflow-pipeline/action.yml

```diff
@@ -106,24 +106,30 @@ runs:
         GCS_TEMPLATE_FILE="gs://${BUCKET_NAME}/templates/${TEMPLATE_FILE_NAME}"
 
         gcloud dataflow flex-template build "${GCS_TEMPLATE_FILE}" \
-          --project "${PROJECT}" \
-          --image "${IMAGE}" \
-          --sdk-language "${PIPELINE_SDK}" \
-          --metadata-file "${TEMPLATE_METADATA_FILE}"
+          --project="${PROJECT}" \
+          --image="${IMAGE}" \
+          --sdk-language="${PIPELINE_SDK}" \
+          --metadata-file="${TEMPLATE_METADATA_FILE}"
 
-        gcloud beta datapipelines pipeline delete "${PIPELINE_NAME}" \
```

Comment:
This is a race condition. It's possible for the pipeline to be created between this call and the next. I think we should always delete, but check the returned error message and ignore it if it's "not found".

Reply:
Changed to ignore failure to delete.